web/src/components/TaskCtaArea.tsx

```diff
@@ -0,0 +1,123 @@
+import { PrimaryButtonDropdown } from "@/components/njwds-extended/PrimaryButtonDropdown";
+import { TaskCTA } from "@/components/TaskCTA";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { PostOnboarding, Task } from "@/lib/types/types";
+import { openInNewTab } from "@/lib/utils/helpers";
+import { ProfileData } from "@businessnjgovnavigator/shared/index";
+import { useRouter } from "next/router";
+import React, { ReactElement } from "react";
+
+interface ButtonsArrayElement {
+  text: string;
+  destination: string;
+  onClick: () => void;
+}
+
+const TaskFooter = (props: React.ComponentPropsWithoutRef<"div">): JSX.Element => {
+  const { children, ...attributes } = props;
+  return (
+    <div
+      data-testid="cta-area"
+      style={{ marginTop: "2rem" }}
+      className="flex flex-justify-end bg-base-lightest margin-x-neg-4 padding-3 margin-bottom-neg-4 flex-column mobile-lg:flex-row radius-bottom-lg"
+      {...attributes}
+    >
+      <div className="mobile-lg:margin-bottom-0 margin-bottom-1">{children}</div>
+    </div>
+  );
+};
+
+interface Props {
+  postOnboardingQuestion: PostOnboarding | undefined;
+  task?: Task | undefined;
+  forceDisplay?: boolean | undefined;
+  onboardingKey?: keyof ProfileData | undefined;
+}
+
+export const TaskCtaArea = (props: Props): ReactElement => {
+  const ctaButtons: ButtonsArrayElement[] = [];
+  const { business } = useUserData();
+  const router = useRouter();
+
+  const taskCtaDestination = props.task?.callToActionLink;
+  let taskCtaText = props.task?.callToActionText ?? "";
+
+  if (taskCtaText && taskCtaText.length === 0) {
+    taskCtaText = "Start Application";
+  }
+
+  const link1 = props.postOnboardingQuestion?.callToActionYesLink1;
+  const text1 = props.postOnboardingQuestion?.callToActionYesText1;
+  const link2 = props.postOnboardingQuestion?.callToActionYesLink2;
+  const text2 = props.postOnboardingQuestion?.callToActionYesText2;
+
+  const hasTaskCta = taskCtaDestination;
+  const hasPostOnboarding = !!props.postOnboardingQuestion;
+
+  const hasAnsweredYesToPostOnboarding =
+    (props.onboardingKey && business?.profileData[props.onboardingKey] === true) || props.forceDisplay;
+
+  if (hasPostOnboarding && hasAnsweredYesToPostOnboarding) {
+    if (link1 && link1.length > 0 && text1 && text1.length > 0) {
```

Comment:
[sand] Same point as above, I believe that checking length is redundant here and this could be simplified to `if (link1 && text1)` which is much more readable :)

Reply:
Same comment on line 74